### PR TITLE
Collaboration cafe notes 20230720-20231003

### PR DESCRIPTION
**Pull request by manics**
- 20230718-usability_and_implementation.md
- 20230801-principles_and_implications.md
- 20230817-evaluating_tres_against_satre.md
- 20230912-governance.md
- 20230921-user_experience_of_the_specification.md
- 20231003-collaboration_cafe.md

I've also added an exported version of the Google Doc used in 20230801-principles_and_implications.md

### CODE REVIEW

File: collaboration_cafes/20230718-usability_and_implementation.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  - This is good, and they should be. the aim is to put them together as they currently are in bits across many docs, policies, reports...
+
+- SDE and TRE as distinct things.
+
+  - What is the difference?
+  - TRE is environment that data comes into, SDE is wider: includes relationship with data provider
+
+- What can we do to ensure that the sub-national SDEs at least are aware of SATRE as possible before they get building, so they can use it for guidance, should they want to?
+
+  - Wataru and Andrew in attendance here
```
Comment (harisood) [inline collaboration_cafes/20230718-usability_and_implementation.md]:
```suggestion

```

File: collaboration_cafes/20230718-usability_and_implementation.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  - Rob Baxter attending NHS community of practice
```
Comment (harisood) [inline collaboration_cafes/20230718-usability_and_implementation.md]:
```suggestion

```

File: collaboration_cafes/20230718-usability_and_implementation.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    - What would make you feel like you have to comply?
+    - You need DSPT for using NHS data, that is mandatory.
+      Researchers will therefore 'demand' this.
+      Operational must-haves are compelling
+      Good alignment with, for example, ISO27001 would make SATRE more attractive.
+      "My driving license lets me drive any model of car"
+    - If SATRE was well aligned, would you be interested in also getting SATRE as an additional achievement?
+      - ISO27001 isn't _for_ research. The fact that SATRE is geared for particular research is a strength.
+        We have ISO27001 for our DSH, having SATRE as a badge of standard for this type of research would be good, reassuring for researchers.
+  - Regarding Claire's idea of recommendations not just requirements. SATRE is also aiming to build TRE implementations which comply with SATRE (TREEHOOSE, Turing DSH, ...). Would this be useful? One reference implementation, many?
```
Comment (harisood) [inline collaboration_cafes/20230718-usability_and_implementation.md]:
```suggestion
  - Regarding idea of recommendations not just requirements. SATRE is also aiming to build TRE implementations which comply with SATRE (TREEHOOSE, Turing DSH, ...). Would this be useful? One reference implementation, many?
```

**Review by harisood — CHANGES_REQUESTED** [review]
Found some names still in text, could you scan through again and remove any names in the actual notes too (I did one or two but then stopped to do my own)

**Comment by manics** [comment]
Done! If you fix the filename typo in https://github.com/sa-tre/satre-notes/pull/3 and merge it I'll rebase this on top and add everything to the index

**Comment by harisood** [comment]
Cool have done @manics !

**Comment by manics** [comment]
Rebased and updated

**Comment by harisood** [comment]
Cool LGTM!

**Pull request merged**